src/main/java/hu/akarnokd/rxjava2/operators/FlowableWindowPredicate.java

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright 2016-2017 David Karnok, Martin Nowak
```

Comment:
Please don't change the library header. You can put your name into the class' `@author` tag.